glide.yaml

```diff
@@ -1,15 +1,20 @@
 package: github.com/openshift/api
 import:
-# this matches the 1.13 branch from kube
+# this matches the 1.14 branch from kube
 - package: k8s.io/gengo
   version: 51747d6e00da1fc578d5a333a93bb2abcbce7a95
 - package: k8s.io/code-generator
-  repo:    git@github.com:openshift/kubernetes-code-generator
-  version: origin-4.1-kubernetes-1.13.4
-# this matches the 1.13.4 GA level of apimachinery which glide isn't wanting to pick
+  repo:    https://github.com/openshift/kubernetes-code-generator
+  version: origin-4.2-kubernetes-1.14.0
```

Comment:
why is this still needed?

Reply:
https://github.com/openshift/kubernetes/commit/dd0fbe2fe552ab7a4600ef126b22c4738c9396e0#diff-ca74ce50776ca750df9347c330f77603 is the reason we need it